tests/client-tests/python-tuf.py

```diff
@@ -25,18 +26,33 @@
     METADATA_DIR = f"{tmpdirname}/metadata"
     os.mkdir(METADATA_DIR)
 
-    # Copy in root metadata to use as trusted root
-    # NOTE: we have to use v5 or newer, once it exists, because prior versions
-    # were not compatible with python-tuf:
+    # Copy in root metadata to use as trusted root.
+    # NOTE: we have to use v5 or newer because prior versions were not
+    # compatible with python-tuf:
     # https://github.com/sigstore/root-signing/issues/103
     # https://github.com/sigstore/root-signing/issues/329
     shutil.copyfile(
-        "repository/repository/4.root.json",
+        "repository/repository/5.root.json",
         f"{METADATA_DIR}/root.json")
 
-    updater = Updater(
-        metadata_dir=METADATA_DIR,
-        metadata_base_url=f"{REPO_URL}/metadata/",
-        target_base_url=f"{REPO_URL}/targets/",
-        target_dir=tmpdirname)
-    updater.refresh()
+    fulcio_cert = "fulcio.crt.pem"
```

Comment:
very small nit: could you add a TODO here saying you're hard-coding a single target. If target retrieval API changes, this would likely change.

Reply:
Absolutely. Addressed in a3d6b450046af5e8274765d11587445b5bab4f07